kafka/librdkafka_vendor/rdkafka.h

```diff
@@ -2056,7 +2056,7 @@ void rd_kafka_conf_set_log_cb(rd_kafka_conf_t *conf,
  *                 rd_kafka_conf_set_opaque().
  *
  * For more information on the format of \p json, see
- * https://github.com/confluentinc/librdkafka/wiki/Statistics
+ * https://github.com/edenhill/librdkafka/wiki/Statistics
```

Comment:
Ideally, this should remain as https://github.com/confluentinc/librdkafka/wiki/Statistics , right?

Reply:
This is taken from the bundle, we should change there first. For the moment let's keep the redirect